docs/nncf_compression.md

```diff
@@ -5,20 +5,12 @@
 OTEDetection allows making compression of the models by
 [NNCF (Neural Network Compression Framework)](https://github.com/openvinotoolkit/nncf/).
 
-The work of OTEDetection is checked with [**NNCF 1.7.0**](https://github.com/openvinotoolkit/nncf/tree/release_v170).
+The work of OTEDetection is validated with [**NNCF 1.7.0**](https://github.com/openvinotoolkit/nncf/tree/release_v170).
```

Comment:
it should be NNCF 1.7.1

Reply:
Sorry, Maxim, but I think it **must NOT** be 1.7.1 here.
The reason is that requirements file `requirements/nncf_compression.txt` contains line
`nncf==1.7.0`
and we cannot change the version of NNCF in documentation, if we do not change it in the requirement file.
And we cannot change the version NNCF in requirements file without additional checks, since this requirement file is used in our CI to install NNCF.

Since now `nncf==2.0.0` has been released, I thought that it is better if somebody will move the version of NNCF to 2.0.0 instead of checking obsolete (already) version 1.7.1

Reply:
It's really strange because 1.7.1 was delivered specifically for OTE. I think we have some misalignment here. cc @vshampor @kshpv 
I think it doesn't stop this particular documentation update so let's merge it.